queries.sql

```diff
@@ -89,9 +113,14 @@ SELECT billing_state, COUNT(*), SUM(total) FROM invoices WHERE billing_country =
 SELECT billing_state, COUNT(*), SUM(total), AVG(total) FROM invoices WHERE billing_country = 'USA' AND billing_state = 'CA' GROUP BY billing_state;
 
 -- The count, total, and average of invoice totals, grouped by state, ordered by average invoice total from highest-to-lowest
+SELECT billing_state, COUNT(*), SUM(total), AVG(totals) FROM invoices WHERE billing_country = 'USA' GROUP BY billing_state ORDER BY AVG(total) DESC;
```

Comment:
Hmm, are you running these queries at all? PostgreSQL can give you much quicker feedback than me when it comes to things like typos, etc. :)